samples/apps/next-native-sample/README.md

```
@@ -0,0 +1,107 @@
+
+| Folder | What it sets up |
+|--------|-----------------|
+| `basic/` | Username and password login — simplest way to get started |
+| `multi-auth/` | Username/password + Google, GitHub, SMS OTP, and Passkey |
+
+### 2. Import ThunderID Resources
+
+Open the `.env` file in your chosen folder and fill in your values:
+
+**`basic/thunderid.env`** — only one value needed:
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)